MNE/utils/ioutils.h

```diff
@@ -254,8 +284,31 @@ bool IOUtils::write_eigen_matrix(const Matrix<T, Dynamic, Dynamic>& in, const QS
 //*************************************************************************************************************
 
 template<typename T>
+bool IOUtils::read_eigen_matrix(Matrix<T, 1, Dynamic>& out, const QString& path)
+{
+    Eigen::Matrix<T, Eigen::Dynamic, Eigen::Dynamic> matrixName(1,out.cols());
+    matrixName.row(0)= out;
+    IOUtils::read_eigen_matrix(matrixName, path);
+}
```

Comment:
Here you need to pay attention to the fact that out is a reference. By adressing the reference to a new variabel matrixName and passing it to read_eigen_matrix you are loosing the reference to out. What you could do is 
`Eigen::Matrix<T, Eigen::Dynamic, Eigen::Dynamic> matrixName;`
`IOUtils::read_eigen_matrix(matrixName, path);`
`if(matrixName.rows() > 0) {`
`out = matrixName.row(0)`
`}`